src/demo/components/blotter/SymbolsRow.tsx

```diff
@@ -83,7 +83,8 @@ export function SymbolsRow(props: SymbolsRowProps): React.ReactElement {
             type: 'fdc3.instrument',
             name: item.name,
             id: {
-                default: item.name
+                ticker: item.ticker,
+                default: item.ticker

```

Comment:
If we're changing it to use `ticker` do we still need `default`? That was just a leftover thing from an old version of the spec right? I don't see it on the fdc3 spec site anymore.

Reply:
Hmm, didn't realise that got removed. Yeah, let's do that.